offline-upgrades/README.md

```diff
@@ -0,0 +1,143 @@
+# Automatic Upgrades in Offline Mode
+
+When trying to use the automatic upgrade functionality provided by the project, you will discover that
+numerous URL as attempted to be accessed when performing automatic upgrades. These URLs are all related

```

Comment:
"...you will discover that numerous URL as attempted to be accessed when performing "reads odd.